pkg/segment/tracing/handler/tracehandler.go

```diff
@@ -155,6 +168,27 @@ func ProcessSearchTracesRequest(ctx *fasthttp.RequestCtx, myid int64) {
 	ctx.SetStatusCode(fasthttp.StatusOK)
 }
 
+func convertTimeToUint64(val interface{}) (uint64, error) {
+	switch v := val.(type) {
+	case float64:
+		return uint64(v), nil
+	case int:
+		return uint64(v), nil
+	case int64:
+		return uint64(v), nil
+	case uint64:
+		return v, nil
+	case string:
+		floatVal, err := strconv.ParseFloat(v, 64)
+		if err != nil {
+			return 0, fmt.Errorf("error converting string to float64: %v", err)
+		}
+		return uint64(floatVal), nil
```

Comment:
wont you need a stconv call here ? 
also write a unit test for this function, passing in various types